renderer/src/webgpu/render_context_webgpu_impl.cpp

```diff
@@ -312,11 +316,11 @@ class RenderContextWebGPUImpl::ColorRampPipeline
         };
 
         wgpu::VertexBufferLayout vertexBufferLayout = {
-            .arrayStride = sizeof(gpu::GradientSpan),
-            .stepMode = wgpu::VertexStepMode::Instance,
             .attributeCount = std::size(attrs),
             .attributes = attrs,
         };
+        vertexBufferLayout.arrayStride = sizeof(gpu::GradientSpan);
```

Comment:
Did you lift these out of the designated initializer because the types didn't match? Would it work with something like:

```
        wgpu::VertexBufferLayout vertexBufferLayout = {
            .arrayStride = static_cast<...>(sizeof(gpu::GradientSpan)),
            .stepMode = static_cast<...>(wgpu::VertexStepMode::Instance),
            .attributeCount = std::size(attrs),
            .attributes = attrs,
```

Reply:
Ah, ok. Makes sense, I don't see a way around it then